include/zephyr/bluetooth/audio/audio.h

```diff
@@ -2018,6 +2018,42 @@ int bt_audio_unicast_group_delete(struct bt_audio_unicast_group *unicast_group);
  * @{
  */
 
+struct bt_audio_broadcast_source_stream_param {
+	/** Audio stream */
+	struct bt_audio_stream *stream;
+
+	/** The number of elements in the @p data array.
+	 *
+	 * The BIS specific data may be omitted and this set to 0.
+	 */
+	size_t data_count;
+
+	/** BIS Codec Specific Configuration */
+	struct bt_codec_data *data;
+};
+
+struct bt_audio_broadcast_source_subgroup_param {
+	/** The number of parameters in @p stream_params */
+	size_t params_count;
+
+	/** Array of stream parameters */
+	struct bt_audio_broadcast_source_stream_param *params;
```

Comment:
It's not entirely clear to me, but is it here it would be possible to set e.g. language metadata as well?

Reply:
The metadata parameters are supplied in the `codec` parameter on the next line, along with the codec configuration parameters